### PR TITLE
ci: move commands in specific section

### PR DESCRIPTION
**Pull request by subhamkrai**
ci: move commands in a specific section

currently, all the commands are in the same
test name, it's better to split the commands
in the individual section.

Signed-off-by: subhamkrai <srai@redhat.com>

### CODE REVIEW

**Comment by travisn** [comment]
LGTM after we get #90 merged

File: .github/workflows/go-test.yaml
Comment (travisn) [inline .github/workflows/go-test.yaml:62]: Even this section, could you break it out to one command per section?

**Review by travisn — CHANGES_REQUESTED** [review]

File: .github/workflows/go-test.yaml
Comment (subhamkrai) [reply .github/workflows/go-test.yaml]: done

**Review by travisn — APPROVED** [review]

**Pull request merged**
